verifier/rest/rest.go

```diff
@@ -109,11 +109,19 @@ func (c *restClient) CreateChallenge(ctx context.Context) (*verifier.Challenge,
 
 // VerifyAttestation implements verifier.Client
 func (c *restClient) VerifyAttestation(ctx context.Context, request verifier.VerifyAttestationRequest) (*verifier.VerifyAttestationResponse, error) {
-	if request.Challenge == nil || request.Attestation == nil {
+	if request.Challenge == nil {
 		return nil, fmt.Errorf("nil value provided in challenge")
 	}
+
+	if request.Attestation == nil && request.TDCCELAttestation == nil {
+		return nil, fmt.Errorf("neither TPM nor TDX attestation is present")
+	} else if request.Attestation != nil && request.TDCCELAttestation != nil {
+		return nil, fmt.Errorf("both TPM and TDX attestation are present, only one should present in the attestation request")
```

Comment:
"both TPM and TDX attestation are present, only one should be present in the attestation request"

Reply:
done